src/routes/MapExplorer/MapExplorer.js

```diff
@@ -90,6 +102,11 @@ export default class MapExplorer extends React.Component {
          !isEqual(prevProps.publicCommunities, publicCommunities))) {
       this.setState(this.updatedMapFeatures(this.state.currentBoundingBox))
     }
+
+    if (prevProps.selectedSearch !== this.props.selectedSearch) {
+      this.updateBoundingBoxQuery(this.props.selectedSearch.boundingBox)
```

Comment:
so this is where it seems like the params would get updated which would automatically call the other update functions to load posts, etc.

Reply:
Good point, I've updated that to use `this.updateSavedSearch` which is also called in `componentDidMount`